x/metadata/keeper/query_server.go

```diff
@@ -155,14 +181,14 @@ func (k Keeper) ValueOwnership(c context.Context, req *types.ValueOwnershipReque
 	return &types.ValueOwnershipResponse{ScopeIds: scopes, Pagination: pageRes}, nil
 }
 
-// Scope returns a specific scope by id
+// ScopeSpecification returns a specific scope by id
 func (k Keeper) ScopeSpecification(c context.Context, req *types.ScopeSpecificationRequest) (*types.ScopeSpecificationResponse, error) {
 	if req == nil {
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
 	if req.SpecificationId == "" {
-		return nil, status.Error(codes.InvalidArgument, "speccification id cannot be empty")
+		return nil, status.Error(codes.InvalidArgument, "specification id cannot be empty")
 	}
 
 	id, err := uuid.Parse(req.SpecificationId)
```

Comment:
Similar issue [granted existing code, not in PR] here re: *_id being treated as a uuid.  We need to be consistent with these.